src/Faker/Provider/lt_LT/Person.php

```diff
@@ -0,0 +1,151 @@
+        "Žemaitis", "Kairys", "Kaminskas", "Mikalauskas", "Petkevičius", "Bagdonas", "Petraitis",
+        "Černauskas", "Vaitkevičius", "Mickevičius", "Urbanavičius", "Jurevičius", "Marcinkevičius",
+        "Pocius", "Paškevičius", "Rutkauskas", "Mackevičius", "Juška", "Malinauskas", "Sinkevičius",
+        "Radzevičius", "Sadauskas", "Kučinskas", "Mažeika", "Vitkauskas", "Vaitkus", "Dambrauskas",
+        "Naujokas", "Sabaliauskas", "Mockus", "Bernotas", "Rakauskas", "Lukoševičius", "Norkus", "Butkevičius",
+        "Barkauskas", "Lukošius", "Kubilius", "Barauskas", "Rinkevičius", "Jonaitis", "Krasauskas", "Vilkas",
+        "Šukys", "Lukaševičius", "Markevičius", "Valaitis", "Mačiulis", "Aleksandravičius", "Venckus",
+        "Grigaliūnas", "Juškevičius", "Baltrušaitis", "Morkūnas", "Tamošiūnas", "Adomavičius", "Stonkus",
+        "Januškevičius", "Lapinskas", "Šidlauskas", "Stankus", "Banys", "Bartkus", "Povilaitis", "Misiūnas",
+        "Rimkevičius", "Mickus", "Karpavičius", "Budrys", "Matulevičius", "Kazakevičius", "Gudaitis", "Račkauskas",
+        "Steponavičius", "Noreika", "Brazauskas", "Grigas", "Vanagas", "Petravičius",
+        "Misevičius", "Miškinis", "Visockas", "Bružas", "Vyšnauskas", "Astrauskas", "Karalius", "Antanavičius",
+        "Šmidtas", "Jakubauskas", "Juknevičius", "Jakštas", "Urbonavičius", "Meškauskas", "Narkevičius", "Tamašauskas",
+        "Kalvaitis", "Vaitiekūnas", "Klimas", "Laurinavičius", "Arlauskas", "Petkus", "Simanavičius", "Šulcas",
+        "Vaičiulis", "Janušauskas", "Vaičiūnas", "Jokubauskas", "Karvelis", "Kasperavičius", "Ališauskas", "Miliauskas",
+        "Giedraitis", "Vitkus", "Gricius", "Stanevičius", "Jurkevičius", "Kulikauskas", "Grigaitis",
+        "Bukauskas", "Šalkauskas", "Jurgaitis", "Linkevičius", "Čepas", "Slavinskas", "Jonušas", "Leonavičius",
+        "Klimavičius", "Katinas", "Ragauskas", "Balsys", "Markūnas", "Jasiūnas", "Lukauskas", "Alekna", "Burba",
+        "Janavičius", "Pečiulis", "Vaškevičius", "Martinkus", "Velička", "Janulevičius", "Peteraitis", "Čepulis",
+        "Blažys", "Kalinauskas", "Petronis", "Poška", "Višniauskas", "Blaževičius", "Cibulskas", "Pocevičius",
+        "Gailius", "Gasiūnas", "Narbutas", "Jablonskas", "Mockevičius", "Norvaišas", "Grigonis", "Lisauskas",
+        "Liutkevičius", "Vilimas", "Masiulis", "Buivydas", "Kriščiūnas", "Liutkus", "Andriuškevičius", "Antanaitis",
+        "Markauskas", "Motiejūnas", "Banevičius", "Bieliauskas", "Kvedaras", "Strazdas", "Žukas", "Gedvilas",
+        "Kriaučiūnas", "Ramonas", "Poškus", "Stonys"
+    );
+
+    protected static $titleMale = array('Dr.');
+
+    protected static $titleFemale = array('Dr.');
+
+    private static $femaleLastNameSuffix = array(
+        'ienė', 'aitė', 'ė'
+    );
+
+    public function lastName()
+    {
+        //get male last name
+        $lastName = static::randomElement(static::$lastName);
+        if (rand(0, 1)) {
```

Comment:
`rand` is forbidden. And this is not the way you should do. Use formats instead.